setup.cfg

```diff
@@ -0,0 +1,67 @@
+[metadata]
+name = nbgrader
+version = attr: nbgrader._version.__version__
+description = A system for assigning and grading notebooks
+long_description = file: README.md
+long_description_content_type = text/markdown
+license_file = LICENSE
+author = Jupyter Development Team
+author_email = jupyter@googlegroups.com
+url = https://github.com/jupyter/nbgrader
+platforms = Linux, Mac OS X, Windows
+keywords = Jupyter, Notebooks, Grading, Homework
+classifiers =
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+
+    Framework :: Jupyter
+    Framework :: Jupyter :: JupyterLab
+    Framework :: Jupyter :: JupyterLab :: 3
+    Framework :: Jupyter :: JupyterLab :: Extensions
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt
```

Comment:
Maybe we should remove these classifiers for now, and add them back when adding the JupyterLab extension?